tutorials/continuous_aggs_tutorial.md

```diff
@@ -0,0 +1,173 @@
+
+# Tutorial: Continuous Aggregates
+
+
+A continuous aggregate query computes the query results automatically at a user specified time interval without the user having to repeatedly rerun the query.  This is particularly useful when recomputing aggregates frequently for large data sets.
+
+We will explore this feature using the data set from the [Hello NYC Tutorial][hello_nyc]
```

Comment:
There is no corresponding link ref at the bottom of the file.

Reply:
fixed